tools/manage-osqueryd.ps1

```diff
@@ -14,11 +14,15 @@ param(
   [switch] $stop = $false,
   
   [switch] $help = $false,
-  [switch] $debug = $false
+  [switch] $debug = $false,
+
+  [switch] $install_wel_manifest = $false,
+  [switch] $uninstall_wel_manifest = $false
```

Comment:
Thoughts on having _one more_ command line switch that's the location of the `osquery.man` file and have it default to the location you expect? That might make it easier for folks to use this script and have custom deployments.